### PR TITLE
Projects and project transfers

### PR DESCRIPTION
**Pull request by labkey-ankurj**

### CODE REVIEW

File: nirc_ehr/resources/domain-templates/ehr.template.xml [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+                    <dat:rangeURI>boolean</dat:rangeURI>
+                </dat:column>
+            </dat:columns>
+        </table>
+    </template>
+
+    <template xsi:type="EHRTemplateType">
+        <table tableName="project" tableDbType="TABLE">
+            <dat:columns>
+                <dat:column columnName="isActive">
```
Comment (labkey-jeckels) [inline nirc_ehr/resources/domain-templates/ehr.template.xml]: Do we need this? Project already has a StartDate and EndDate column which are often enough to do a calculated isActive field.

File: nirc_ehr/resources/schemas/dbscripts/postgresql/nirc_ehr-21.002-21.003.sql [hunk 2]
```diff
@@ -0,0 +1,3 @@
+SELECT core.executeJavaUpgradeCode('reloadStudy');
+SELECT core.executeJavaUpgradeCode('etl;{NIRC_EHR}/project');
```
Comment (labkey-jeckels) [inline nirc_ehr/resources/schemas/dbscripts/postgresql/nirc_ehr-21.002-21.003.sql:2]: This is fine to leave here, but until we have a mechanism to reload the extensible table columns they'll still need to rerun the ETL after manually creating the columns

**Review by labkey-jeckels — APPROVED** [review]

File: nirc_ehr/resources/schemas/dbscripts/postgresql/nirc_ehr-21.002-21.003.sql (hunk 2 above)
Comment (labkey-ankurj) [reply nirc_ehr/resources/schemas/dbscripts/postgresql/nirc_ehr-21.002-21.003.sql:2]: I had it in my list to remove the extensible defn for project that I missed as now I am not pulling those columns from the different table after getting clarification yesterday. I think you caught 'isActive' column as well in extensible defn, I have removed the whole defn for project so need not to worry about the reload the defn in this commit but def a good thing for future auto re-load lists.

File: nirc_ehr/resources/domain-templates/ehr.template.xml (hunk 1 above)
Comment (labkey-ankurj) [reply nirc_ehr/resources/domain-templates/ehr.template.xml]: nope, removed the full defn as I don't need the other column as well.

**Pull request merged**